about/extensions.html.md.erb

```diff
@@ -0,0 +1,58 @@
+---
+title: Fly.io Extensions Program
+layout: docs
+sitemap: true
+nav: firecracker
+---
+
+Fly.io Extensions Program
+
+Fly.io is a global cloud service for running full stack apps close to users. We're committed to giving customers a great developer experience, and we're building a platform that makes it easy to deploy and manage apps in any region of the world, beyond the footprint of traditional cloud platforms.
+
+Extensions are 3rd party services  -- those that we wouldn't build ourselves -- that extend our platform to serve customer needs. Services such as managed databases, exception handlers or log aggregators fit the bill. Check out our [managed Redis extension by Upstash](https://fly.io/docs/reference/redis/) to get an idea of what's possible.
+
+Now, rather than bolting a grab bag of products on to ours, we're interested in deep integration with a few select partners. We want to offer a seamless experience for customers, and we want to make it easy for you to reach them.
+
+We run our own hardware, all over the world, and we're cheaper than big clouds. [Check out our documentation](https://fly.io/docs) to see what our platform has to offer.
```

Comment:
Are we? We're cheaper than clouds that build on other clouds...or is AWS not a cloud?

Reply:
We are, but we maybe should not say it this way. I'll correct.